python/schemas/v2/Mapping.schema

```diff
@@ -215,36 +218,18 @@
                 "additionalProperties": false
             }
         },
-        "generation": {
-            "type": "integer"
-        },
-        "grpc": {
-            "type": "boolean"
-        },
-        "headers": {
-            "$ref": "#/definitions/mapStrStr"
-        },
-        "host": {
-            "type": "string"
-        },
-        "host_redirect": {
-            "type": "boolean"
-        },
-        "host_regex": {
-            "type": "boolean"
-        },
-        "host_rewrite": {
-            "type": "string"
-        },
-        "hostname": {
-            "type": "string"
-        },
-        "idle_timeout_ms": {
-            "type": "integer"
-        },
-        "kind": {
-            "type": "string"
+        "modules": {
+            "type": "array",
+            "items": {
+                "type": "object"
+            }
         },
+        "host": { "type": "string" },
+        "host_regex": { "type": "boolean" },
+        "hostname": { "type": "string" },
+        "headers": { "$ref": "#/definitions/mapStrStr" },
+        "regex_headers": { "$ref": "#/definitions/mapStrStr" },
+        "respect_dns_ttl": { "type": "boolean" },
```

Comment:
It looks like you didn't add `respect_dns_ttl` to `pkg/api/getambassador.io/v3alpha1/mapping_types.go`?

Reply:
`make lint` is unhappy that this doesn't match `make format`.